test/extensions/filters/network/common/fuzz/network_filter_fuzz.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package test.extensions.filters.network;
+import "google/protobuf/empty.proto";
+import "validate/validate.proto";
+import "envoy/config/listener/v3/listener_components.proto";
+
+message OnData {
+  bytes data = 1;
+  bool end_stream = 2;
+}
+
+message AdvanceTime {
+  uint32 milliseconds = 1 [(validate.rules).uint32 = {gt: 0}];
+}
```

Comment:
Should there be an upper limit?  An hour seems like a sane upper limit on the amount time would advance in one increment.

Reply:
> Should there be an upper limit? An hour seems like a sane upper limit on the amount time would advance in one increment.

Added!